docs/quickstart.md

```diff
@@ -10,7 +10,7 @@ This is a guide to getting started with using Axis using Minikube.
 
 ## 1. Checkout project's master branch
 ```
-$ git clone git@github.com:/axis.git
+$ git clone git@github.com:argoproj/argo-events.git blackrock/axis
```

Comment:
do we need `blackrock/axis` appended after clone?

Reply:
Done.